### PR TITLE
Gp2 3164- Updating vulnerability for great-cms

### PR DESCRIPTION
**Pull request by seijihg**
Updating packages for vulnerability.

Main great-cms:
Screenshot 2021-08-16 at 22 56 07
 
### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/GP2-3164
- [x] Jira ticket has the correct status.
- [ ] [Changelog](CHANGELOG.md) entry added.

### Reviewing help

- [ ] Explains how to test locally, including how to set up appropriate data
- [x] Includes screenshot(s) - ideally before and after, but at least after

### Housekeeping

- [ ] Added all new environment variables to Vault.
- [ ] Cleaned up old feature flags
- [x] Upgraded any vulnerable dependencies.
- [ ] Ran the `make manage download_geolocation_data` command
- [x] I have updated security dependencies
- [ ] Python requirements have been re-compiled.
- [x] Frontend assets have been re-compiled.

### Merging

- [x] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

**Comment by froddd** [comment]
First file in the changes view doesn't look too right: `core/components/. babelrc` (there's a space after the dot?!)

File: core/components/babel.config.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@babel/preset-env", "@babel/preset-react"]
+}
```
Comment (froddd) [inline core/components/babel.config.json:3]: There's a slight precedent for using runcom files for front-end config, so may be worth putting this in `.babelrc` (which is what the above file may have attempted to do!)

File: core/components/static/javascript/reactComponents.js.LICENSE.txt [hunk 2]
```diff
@@ -0,0 +1,50 @@
+/*
```
Comment (froddd) [inline core/components/static/javascript/reactComponents.js.LICENSE.txt:1]: Probably can remove this file, or is there a way to not generate it?

File: core/components/static/stylesheets/elements-components.css [hunk 3]
```diff
@@ -4128,6 +4128,9 @@ section.blue-underline > .container {
   margin: 0 auto;
   float: none; }
 
+.relative {
```
Comment (froddd) [inline core/components/static/stylesheets/elements-components.css:4131]: Is this a source file? Are these changes required as part of this?

File: react-components/dist/magna.js.LICENSE.txt [hunk 4]
```diff
@@ -0,0 +1,72 @@
+/*
```
Comment (froddd) [inline react-components/dist/magna.js.LICENSE.txt:1]: Same as above, probably could do without this!

**Review by froddd — COMMENTED** [review]
There's a few font `.svg` files that have ended up in the images directory -- a fix for that has been merged already, you should be able to pull in develop and rerun build to fix.

File: core/components/static/stylesheets/elements-components.css (hunk 3 above)
Comment (seijihg) [reply core/components/static/stylesheets/elements-components.css:4131]: all these got build from  shared/_base.scc

they were changed there on develop but gulp build was never run so when i run it these css got updated.

File: core/components/gulpfile.js [hunk 5]
```diff
@@ -110,7 +43,7 @@ gulp.task(
       )
       .pipe(gulp.dest('static/stylesheets'))
       .pipe(rename({ suffix: '.min' }))
-      .pipe(cssnano())
+      .pipe(cleanCSS({compatibility: 'ie8'}))
```
Comment (froddd) [inline core/components/gulpfile.js:46]: I don't think I've seen IE8 mentioned for a few years now... If this is required can we add a comment specifying why?

**Review by froddd — APPROVED** [review]
This looks good to go
